src/dewey/config.clj

```diff
@@ -62,6 +62,11 @@
   [props config-valid configs]
   "dewey.amqp.qos" 100)
 
+(cc/defprop-optstr es-uri
+  "The hostname for the Elasticsearch server"
+  [props config-valid configs]
+  "dewey.es.uri" "http://elasticsearch:9200")
+
 (cc/defprop-optstr es-host
```

Comment:
Does it make sense to keep around these old config properties? I'm guessing they aren't used any more with the switch to using the URI.